UI/forms/OBSAbout.ui

```diff
@@ -6,36 +6,39 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>840</width>
-    <height>519</height>
+    <width>650</width>
+    <height>400</height>
```

Comment:
I would probably vote to make it taller, to make the content easier to read. At least 500.